src/main/java/com/thinkful/noteful/auth/JwtAuthorizationFilter.java

```diff
@@ -0,0 +1,63 @@
+package com.thinkful.noteful.auth;
+
+import static com.thinkful.noteful.auth.SecurityConstants.HEADER_NAME;
+import static com.thinkful.noteful.auth.SecurityConstants.TOKEN_PREFIX;
+import static com.thinkful.noteful.auth.SecurityConstants.JWT_SECRET;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
`import org.springframework.beans.factory.annotation.Autowired;` - looks like it can be dropped.

Reply:
Done